pyglossary/os_utils.py

```diff
@@ -85,16 +85,17 @@ def _dictzip(filename: str) -> bool:
 	dictzipCmd = shutil.which("dictzip")
 	if not dictzipCmd:
 		return False
-	b_out, b_err = subprocess.Popen(
-		[dictzipCmd, filename],
-		stdout=subprocess.PIPE).communicate()
 	log.debug(f"dictzip command: {dictzipCmd!r}")
-	if b_err:
-		err = b_err.decode("utf-8").replace('\n', ' ')
-		log.error(f"dictzip error: {err}")
-	if b_out:
-		out = b_out.decode("utf-8").replace('\n', ' ')
-		log.error(f"dictzip error: {out}")
+	try:
+		subprocess.run(
+			[dictzipCmd, filename],
+			check=True,
+			stdout=subprocess.PIPE,
+			stderr=subprocess.STDOUT)
+	except subprocess.CalledProcessError as proc_err:
+		err_msg = proc_err.output.decode("utf-8").replace("\n", ";")
+		retcode = proc_err.returncode
+		log.error(f"dictzip exit {retcode}: {err_msg}")
```

Comment:
return False

Reply:
The function follows initial `runDictzip()` behavior on error: write to log and succeed. `False` is only for missing dependency to drop a function in a chain.